src/features/farming/crops/components/RemoveCrop.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Modal } from "react-bootstrap";
+import { Panel } from "components/ui/Panel";
+import { Button } from "components/ui/Button";
+
+import close from "assets/icons/close.png";
+import rustyShovel from "assets/tools/rusty_shovel.png";
+
+interface Props {
+  doShow: boolean;
+  onConfirm: () => void;
+  onClose: () => void;
+}
+
+export const RemoveCrop: React.FC<Props> = ({ doShow, onConfirm, onClose }) => {
+  return (
+    <Modal centered show={doShow} onHide={onClose}>
+      <Panel>
+        <img
+          src={close}
+          className="h-6 top-4 right-4 absolute cursor-pointer"
+          onClick={onClose}
+        />
+        <div className="p-2">
+          <h1 className="text-xl text-center">Remove Growing Crops?</h1>
+          <div className="flex my-4 justify-center">
+            <img src={rustyShovel} style={{ width: "50px" }} />
+          </div>
+          <p className="text-sm mb-4">
+            Warning! You have your shovel equipped.
```

Comment:
I think this would read better as `Your shovel is equipped!` Let the exclamation point be the warning.